examples/oauth2/README.md

```diff
@@ -0,0 +1,8 @@
+# Token Generation
+
+## Enviroment
+
+`INDYKITE_APPLICATION_CREDENTIALS_FILE`
```

Comment:
`DefaultEnvironmentLoader` supports also `INDYKITE_APPLICATION_CREDENTIALS`
Where you specify credentials directly inside that Env var